### PR TITLE
feat: Sass

### PR DESCRIPTION
**Pull request by raulwwq0**
Sass has been installed!!!

Things I did:
- Install Sass and create an ITCSS folder structure
- Adapt our old CSS styles to Sass using SCSS syntax
- Add a little guide in README.md
- Re-enable Github Actions

### CODE REVIEW

File: assets/styles/main.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@use 'sass/01-generic/generic';
+@use 'sass/02-components/header';
+@use 'sass/02-components/scroll-indicator';
+@use 'sass/03-utils/header-scroll-animation';
```
Comment (ImOnlyYisus) [inline assets/styles/main.scss:4]: Can you import reset css from package reset-css?, I think that reset sass file is in diferent path on node_modules

**Review by ImOnlyYisus — CHANGES_REQUESTED** [review]

File: assets/styles/main.scss (hunk 1 above)
Comment (raulwwq0) [reply assets/styles/main.scss:4]: Yes sir

**Review by ImOnlyYisus — APPROVED** [review]
LGTM! 👍

**Review by 00IRCA — APPROVED** [review]
Awesome!

**Pull request merged**